README.md

```diff
@@ -0,0 +1,50 @@
+# BigInt
+
+## Description
+
+This crate is an implementation of a big integer type. It is based on GMP and
+thus offers both the speed and precision this library allows. Its aim is to
+provide an easy to use `BigInt` type that can be used in any crypto crate.
+
+Since efficiency is often critical in cryptographic computations, this crate
+also provides an RNS representation of integers ([see
+below](#RNS)).
+
+## GMP
+
+The [GNU Multi Precision](https://gmplib.org/) arithmetic library is a portable library written in C.
+It provides representations and arithmetic operations over integers, rational
+number and floating points [3]. This crate focuses on the `mpz` integers, even
+though it could easily be extended.
+
+In order to compile this crate, on has to install GMP. This can be done by
+installing the dev package provided by the package manager on any recent Linux
+system, or by installing it from source.
```

Comment:
```suggestion
In order to compile this crate, one has to install GMP. This can be done by
installing the dev package `libgmp-dev` provided by the package manager on any recent Linux
system, or by installing it from source.
```

Reply:
Are we sure this will be this exact name on all Linux distributions ?

Reply:
You are right, I renamed from `libgmp-dev` to `gmp-dev`, which covers Arch and Ubuntu/Debian at least.